### PR TITLE
Run CI once in PR-s from local branches

### PR DESCRIPTION
**Pull request by szepeviktor**
This way it will run only once in a PR from a local branch.

Live example: https://github.com/szepeviktor/byte-level-care/pull/91

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,18 +16,13 @@ env:
...
   test:
     name: Rust ${{matrix.rust}}
-    needs: pre_ci
-    if: needs.pre_ci.outputs.continue
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        rust: [1.56.0, stable, beta]
+        rust: ["1.56.0", stable, beta]
```
Comment (szepeviktor) [inline .github/workflows/ci.yml:25]: YAML understand `1.10` as a float = 1.1

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -2,9 +2,12 @@ name: CI
 
 on:
   push:
+    branches:
+      - master
   pull_request:
   workflow_dispatch:
-  schedule: [cron: "40 1 * * *"]
+  schedule:
+    - cron: "40 1 * * *"
```
Comment (szepeviktor) [inline .github/workflows/ci.yml:10]: Compressing YAML this much is bad practice.

**Comment by szepeviktor** [comment]
Oh no!
You want to run CI on the push event, not on the pull_request event.

image

**Review by dtolnay — COMMENTED** [review]
> Oh no!
> You want to run CI on the push event, not on the pull_request event.

Yeah I run it on push, and it's done by the time I finish writing the PR summary.

Thanks anyway!

**Pull request closed without merging**